_includes/manuals/1.8/4.3.5.3_ms_dhcp.md

```diff
@@ -5,131 +5,28 @@ It is required that this procedure is executed as an administrator.
-  gem install --version 0.43 --platform x86-mingw32 linecache
-  gem install --version 1.16 --platform x86-mingw32 mime-types
-  gem install --version 0.9.11 --platform x86-mingw32 mocha
-  gem install --version 1.3.7 --platform x86-mingw32 net-ping
-  gem install --version 1.2.0 --platform x86-mingw32 rack
-  gem install --version 0.8.7 --platform x86-mingw32 rake
-  gem install --version 1.6.1 --platform x86-mingw32 rest-client
-  gem install --version 1.1.0 --platform x86-mingw32 sinatra
-  gem install --version 1.1 --platform x86-mingw32 tilt
-  gem install --version 1.4.6 --platform x86-mingw32 win32-api
-  gem install --version 0.3.2 --platform x86-mingw32 win32-open3
-  gem install --version 0.7.1 --platform x86-mingw32-60 win32-service
-  gem install --version 0.4.0 --platform x86-mingw32 windows-api
-  gem install --version 1.1.2 --platform x86-mingw32 windows-pr
-</pre>
-
-Command to download them all:
-
-<pre>
-wget http://rubygems.org/downloads/columnize-0.3.2.gem \
-     http://rubygems.org/downloads/haml-3.0.24.gem \
-     http://rubygems.org/downloads/highline-1.6.1.gem \
-     http://rubygems.org/downloads/json-1.4.6-x86-mingw32.gem \
-     http://rubygems.org/downloads/linecache-0.43-mswin32.gem \
-     http://rubygems.org/downloads/mime-types-1.16.gem \
-     http://rubygems.org/downloads/mocha-0.9.11.gem \
-     http://rubygems.org/downloads/net-ping-1.3.7.gem \
-     http://rubygems.org/downloads/rack-1.2.0.gem \
-     http://rubygems.org/downloads/rake-0.8.7.gem \
-     http://rubygems.org/downloads/rest-client-1.6.1.gem \
-     http://rubygems.org/downloads/sinatra-1.1.0.gem \
-     http://rubygems.org/downloads/tilt-1.1.gem \
-     http://rubygems.org/downloads/win32-api-1.4.6-x86-mingw32.gem \
-     http://rubygems.org/downloads/win32-open3-0.3.2-x86-mingw32.gem \
-     http://rubygems.org/downloads/win32-service-0.7.1-x86-mswin32-60.gem \
-     http://rubygems.org/downloads/windows-api-0.4.0.gem \
-     http://rubygems.org/downloads/windows-pr-1.1.2.gem
-</pre>
-
-8) CD to the root of the smart-proxy install directory
-
-9) Edit config/settings.yml so that it looks a bit like this
+1. Go to the smart-proxy repository at https://github.com/theforeman/smart-proxy/releases
+2. Just under the tag matching your Foreman version, click the ZIP link in order to download the Smart Proxy source code
+3. Extract the archive to a directory that does not have any spaces in its name
+4. Go to the rubyinstaller web page at http://rubyinstaller.org/downloads/
+5. Download and install the "ruby 1.9.3-p551": (Allow the ruby associations to be installed.)
+6. Download and install the Development Kit For Ruby 1.9.3: (DevKit-tdm-32-4.5.2-20111229-1559-sfx.exe)
+7. Open a CMD window and, install the bundler_ext gem using *gem install --platform x86-mingw32 bundler_ext*
+8. CD to the root of the smart-proxy directory (the one you created in step 3)
+9. Copy the file Gemfile to GemFile.in : *copy Gemfile GemFile.in*
```

Comment:
I think you can leave this step out, and shouldn't need to install bundler_ext - just bundler.

On step 10 you use bundler, which should be the right way of doing it, we tend to only use bundler_ext when packaging gems as debs/rpms.